frontend/src/dashboard.js

```diff
@@ -361,6 +361,16 @@ export class Dashboard extends React.Component {
                         onEditClick={() => this.onEditWidgetClick(widget.id)}
                       />
                     }
+                    {(widget.type === "widget" && widget.widget === "filter-heatmap") &&
+                      <JenkinsHeatmapWidget
```

Comment:
We should actually switch these around, convert the `JenkinsHeatmapWidget` to a `FilterHeatmapWidget` and then make the Jenkins heatmap use the `FilterHeatmapWidget`